src/Widgets/AppContextMenu.vala

```diff
@@ -65,6 +71,24 @@ public class Slingshot.AppContextMenu : Gtk.Menu {
             });
         }
 
+        if (switcheroo_control != null && switcheroo_control.has_dual_gpu) {
+            bool prefers_non_default_gpu = app_info.get_boolean ("PrefersNonDefaultGPU");
+
+            string gpu_name = switcheroo_control.get_gpu_name (prefers_non_default_gpu);
+
+            string label = _("Open using %s Graphics Card").printf (gpu_name);
```

Comment:
Now I'm just nitpicking, but this is shorter and still reads naturally to me. Plus the idea of an Intel graphics “card” seems funny to me when it's integrated. 😅

```suggestion
            string label = _("Open with %s Graphics").printf (gpu_name);
```